sdk/identity/Azure.Identity/tests/IdentityRecordedTestSanitizer.cs

```diff
@@ -18,62 +15,23 @@ public IdentityRecordedTestSanitizer()
             SanitizedHeaders.Add("secret");
             AddJsonPathSanitizer("$..refresh_token");
             AddJsonPathSanitizer("$..access_token");
-        }
-
-        public override void Sanitize(RecordEntry entry)
-        {
-            if (entry.RequestUri.Split('?')[0].EndsWith("/token"))
+            BodyRegexSanitizers.Add(new BodyRegexSanitizer(".+", SanitizeValue)
```

Comment:
Does the make the two sanitizers above redundant?

Reply:
No, because this sanitizer is restricted to requests that have a requestUri ending with `token`